docs/book/src/reference/metrics.md

```diff
@@ -69,6 +70,43 @@ NAME                                         AGE
 monitor-controller-manager-metrics-monitor   2m8s
 ```
 
+3. If you are using the suggested
+[kube-prometheus](https://github.com/coreos/kube-prometheus#installing) as a
+monitoring system, and you want Prometheus Operator to have access to the
+controller metrics, create the necessary RBAC resources by applying the
+following `Role` and `RoleBinding` for Prometheus to be able to monitor our
+controller namespace.
```

Comment:
```suggestion
<aside class="warning">
<h2>If you are using Prometheus Operator ensure that you have the required permissions </h2>

If you are using [Prometheus Operator][rbac-promethues-operator] then be aware that by default, its RBAC rules are only enabled for the Default and kube-system namespaces. See
its documentation about how to configure the [kube-prometheus](https://github.com/prometheus-operator/kube-prometheus/blob/main/docs/monitoring-other-namespaces.md) to monitor another
namespaces with `.jsonnet`.

However, you can also enable it using RBAC. See the Prometheus Operator [Getting Started](https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/user-guides/getting-started.md#enable-rbac-rules-for-prometheus-pods) documentation to know how to enable the permissions on the namespace where the ServiceMonitor and manager are applied.
</aside>

Reply:
@machadovilaca,

(imo) we need to link here the Prometheus Operator documentation.
See that it will help the users know more about if they need it as well.

On top of that, `<aside class="warning">`  will create a note/CSS on the docs that can retain the attention of the users.